catalog/MDCCatalog.xcodeproj/xcshareddata/xcschemes/MDCCatalog.xcscheme

```diff
@@ -715,31 +731,43 @@
             skipped = "NO">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "80FDCBA97111C2F0E5D54C41D9C997FC"
+               BlueprintIdentifier = "4DA89912E26C1D30834DA796EADFD68E"
                BuildableName = "MaterialComponents-Unit-BottomNavigation+Theming-UnitTests.xctest"
                BlueprintName = "MaterialComponents-Unit-BottomNavigation+Theming-UnitTests"
                ReferencedContainer = "container:Pods/Pods.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "8E9B17CEB9CECE59A5FEEB2347DF98BE"
+               BuildableName = "MaterialComponents-Unit-ProgressView+Theming-UnitTests.xctest"
+               BlueprintName = "MaterialComponents-Unit-ProgressView+Theming-UnitTests"
+               ReferencedContainer = "container:Pods/Pods.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
```

Comment:
Looks like this was added by mistake?

Reply:
I just added all the unit tests targets that weren't already there.